randovania/game_description/pretty_print.py

```diff
@@ -173,7 +173,8 @@ def write_human_readable_meta(game: GameDescription, output: TextIO) -> None:
 
     output.write("\n====================\nDock Weaknesses\n")
     for dock_type in game.dock_weakness_database.dock_types:
-        output.write(f"\n> {dock_type.long_name}")
+        output.write(f"\n> {dock_type.long_name}; "
+                     f"Needs to connect to same type? {dock_type.needs_to_connect_to_same_type}")
```

Comment:
Could only show the "Needs to ..." text when true, and remove the `?` from it.

Reply:
I thought about it too, but personally felt it better to read if i always know the status of it, rather than false unless stated otherwise.